integreat_cms/cms/models/pois/poi.py

```diff
@@ -1,15 +1,14 @@
-from django.db import models
 from django.core.validators import MaxValueValidator, MinValueValidator
+from django.db import models
 from django.utils.translation import gettext_lazy as _
-
 from linkcheck.models import Link
 
+from ...utils.translation_utils import gettext_many_lazy as __
```

Comment:
I'm not completly understanding why this import is on top of the list, since it's not in alphabetical order :thinking:

Reply:
There are three categories which are more important than the alphabetical order:
1. Imports from the python standard library go on top (e.g. `logging`, `os`, `sys`, etc.)
2. Imports from third-party packages go second (`django`, `linkcheck`, etc.)
3. Relative imports go third (`from ...x import y` etc.)

Reply:
Yes, that I understand. I was just wondering why it was on top of the relative imports, since within the block of relative imports it wasn't ordered alphabetically. But seems to be resolved now anyway!

Reply:
Ah ok, yeah within the relative imports, the leading dots are counted as well. So
`..b` is sorted before `.a` and so on...

Reply:
So imports from "far away packages" are sorted first, and then as we descend the list we get nearer to the current code :)

Reply:
Okay, that's explains it. Thank you very much!